Pipfile

```diff
@@ -16,7 +16,7 @@ requests-mock = "*"
 
 [packages]
 citrine = {editable = true,path = "."}
-taurus = {git = "https://github.com/CitrineInformatics/taurus.git", editable = true, ref="develop"}
+gemd = {git = "https://github.com/CitrineInformatics/gemd-python.git", editable = true, ref="develop"}
```

Comment:
It looks like gemd doesn't have a develop branch. Did it used to have one? In that case I think it should be changed to `ref="master"`

Reply:
Yeah, it got moved into masterflow as well.  Good catch.

Reply:
```suggestion
gemd = {git = "https://github.com/CitrineInformatics/gemd-python.git", editable = true, ref="master"}
```